### PR TITLE
Fix the CLOS service to trigger on CLOS updates

### PR DESCRIPTION
**Pull request by FinnStutzenstein**
additionally fixed some naming issues

### CODE REVIEW

File: client/src/app/site/projector/services/current-list-of-speakers.service.ts [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+    /**
+     * Active subscriptions for the clos of each projector.
+     */
+    private currentListOfSpeakersSubscriptions: { [projectorId: number]: Subscription } = {};
+
+    /**
+     * All subjects for all clos of each projector.
+     */
+    private currentListOfSpeakers: { [projectorId: number]: BehaviorSubject<ViewListOfSpeakers | null> } = {};
```
Comment (tsiegleauq) [inline client/src/app/site/projector/services/current-list-of-speakers.service.ts:35]: please add a public getter for an observable to the current list of speakers
`public get currentListOfSpeakers(): Observable<ViewListOfSpeakers>`

no reference id, no projector

Comment (FinnStutzenstein) [reply client/src/app/site/projector/services/current-list-of-speakers.service.ts:35]: We do have just one reference projector, right?

I guess the code comes from the time with multiple ones ;)

Comment (tsiegleauq) [reply client/src/app/site/projector/services/current-list-of-speakers.service.ts:35]: there was an idea to add multiple reference projectors and multiple list of speakers. It was later discarded

Comment (FinnStutzenstein) [reply client/src/app/site/projector/services/current-list-of-speakers.service.ts:35]: Not only the idea, but we *had* this already implemented what then was discarded (except for this service) ;)

**Pull request merged**